userspace/falco/outputs.h

```diff
@@ -51,6 +51,7 @@ struct message
 	std::string source;
 	map<std::string, std::string> fields;
 	std::set<std::string> tags;
+	gen_event_formatter::output_format format;
```

Comment:
Could we avoid passing the formatter in the message?

This approach cannot work with `internal` messages, that do not have a formatter :point_down: 
https://github.com/falcosecurity/falco/blob/226d1fb7284319b5dfb7cbed50281ca11cfda7ad/userspace/falco/falco_outputs.cpp#L199-L213

Moreover, the JSON formatting option is a global setting (not a per-message setting) and the message already contains the formatted text. So I believe the formatter does not really belong to the message.

I would rather pass the `m_json_output`'s value to each output via the `init` function :point_down: 
https://github.com/falcosecurity/falco/blob/226d1fb7284319b5dfb7cbed50281ca11cfda7ad/userspace/falco/falco_outputs.cpp#L138

Reply:
Much simpler, you're right! Thanks ;)